Godeps/_workspace/src/github.com/golang/glog/glog.go

```diff
@@ -876,7 +876,7 @@ const flushInterval = 30 * time.Second
 
 // flushDaemon periodically flushes the log file buffers.
 func (l *loggingT) flushDaemon() {
-	for _ = range time.NewTicker(flushInterval).C {
```

Comment:
Amazingly it looks like the Go authors don't run `go fmt -s` ... this was an automated change, I'll revert